Sources/BraintreeSEPADirectDebit/SEPADebitAccountsRequest.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// The POST body for `v1/payment_methods/sepa_debit_accounts`
+struct SEPADebitAccountsRequest: Encodable {
+
+    private let createMandateResult: SEPADebitAccountRequest
+
+    enum CodingKeys: String, CodingKey {
+        case createMandateResult = "sepa_debit_account"
+    }
+
+    init(createMandateResult: CreateMandateResult) {
+        self.createMandateResult = SEPADebitAccountRequest(
+            last4: createMandateResult.ibanLastFour,
+            merchantOrPartnerCustomerID: createMandateResult.customerID,
+            bankReferenceToken: createMandateResult.bankReferenceToken,
+            mandateType: createMandateResult.mandateType
+        )
+    }
+
+    struct SEPADebitAccountRequest: Encodable {
+
+        let last4: String?
+        let merchantOrPartnerCustomerID: String?
+        let bankReferenceToken: String?
+        let mandateType: String?
```

Comment:
nit(entirely optional): maybe we can add short in-line docs to describe each property within these structs